src/compressed_tensors/entrypoints/convert/converters/modelopt_nvfp4.py

```
@@ -130,17 +130,15 @@ def get_dependencies(self, weight_name: str) -> set[str]:
-                "config_group_0": QuantizationScheme(
-                    **NVFP4,
-                    targets=self.targets,
-                    format=CompressionFormat.nvfp4_pack_quantized.value,
-                )
-            },
-            ignore=self.ignore,
-            kv_cache_scheme=self.kv_cache_scheme,
-            format=CompressionFormat.nvfp4_pack_quantized.value,
-            quantization_status=QuantizationStatus.COMPRESSED.value,
+    def update_config(self, config: dict) -> dict:
+        merge_quantization_config(
```

Ruff (0.15.12)

[error] Undefined name `merge_quantization_config`

(F821)